dashboard/v1.1/src/pages/Dashboard/CPUUsage.tsx

```diff
@@ -54,6 +53,9 @@ const CPUUsage: FC<CPUUsageProps> = ({ cpuMetrics }) => {
         opacityFrom: 1,
         opacityTo: 0.2
       }
+    },
+    theme: {
+      mode: localStorage.getItem('dark-mode') === 'true' ? 'dark' : 'light'
```

Comment:
Let's not reference localStorage in every case, what I'd say is to use localStorage once(if you can) in any top-level component and pass it as a context variable. The issue is that the read performance of localStorage can be costly at times. I've noticed my browser freezing on this branch and referencing localStorage constantly can be a cause. Also if all the charts have the same theme prop, what about instantiating it once and referencing it everywhere.